ui-libraries/material/docs/src/content/docs/getting-started.mdx

```diff
@@ -14,84 +14,84 @@ import { Steps, Tabs, TabItem } from '@astrojs/starlight/components';
 Welcome to the documentation for the Material 3 Design System for Slint. Learn more about
 Slint [here](https://slint.dev/).
 
-We recommend using an IDE for development, along with our [LSP-based IDE integration for
-.slint files](https://github.com/slint-ui/slint/blob/master/tools/lsp/README.md). You
-can also load this project directly in [Visual Studio Code](https://code.visualstudio.com/)
-and install our [Slint extension](https://marketplace.visualstudio.com/items?itemName=Slint.slint).
+If you are starting a project from scratch, you can follow the instructions from the one of our template reprositories:
 
-<Tabs>
+<Tabs syncKey="dev-language">
+<TabItem label="C++">
+https://github.com/slint-ui/material-cpp-template
+</TabItem>
 <TabItem label="Rust">
+https://github.com/slint-ui/material-rust-template
+</TabItem>
+<TabItem label="NodeJS">
+https://github.com/slint-ui/material-nodejs-template
+</TabItem>
+<TabItem label="Python" >
+https://github.com/slint-ui/material-python-template
+</TabItem>
+</Tabs>
 
-1. Install Rust by following its [getting-started guide](https://www.rust-lang.org/learn/get-started). Once this is done, you should have the rustc compiler and the cargo build system installed in your `PATH`.
-2. Clone the getting-started repository:
-```bash
-git clone --recurse-submodules https://github.com/slint-ui/material-rust-template.git my-project
-cd my-project
-```
-3. Build the sample application:
-```bash
-cargo build
-```
-4. Run the sample application:
-```bash
-cargo run
-```
+If you are want to use the components in your existing project, you can download the source code of the material component from
+https://material.slint.dev/zip/material-1.0.zip
+and extract it into your repository.
```

Comment:
Remove conditionals, as in practice this is what you need to do (section 2 of the writing style guide)

```suggestion
To use the components in your existing project, download the source code of the material component from
https://material.slint.dev/zip/material-1.0.zip
and extract it into your repository.
```